Okta.Xamarin/Okta.Xamarin.Android/AndroidOidcClient.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Okta.Xamarin
 {
-	public partial class OidcClient
+	public class AndroidOidcClient : OidcClient
```

Comment:
I wonder if it makes sense to have an Android-specific class in the Okta.Xamarin namespace? What's the motivation here?

Reply:
This was to unravel the partial class links used in the platform specific projects.  I probably should change the namespace, I'll note that for a follow up and add it to my latest working branch.